lerobot/common/logger.py

```diff
@@ -114,6 +114,8 @@ def log_dict(self, d, step, mode="train"):
         assert mode in {"train", "eval"}
         if self._wandb is not None:
             for k, v in d.items():
+                if not isinstance(v, (int, float, str)):
+                    continue
```

Comment:
This silent `continue` may cause frustration when logged values are silently dropped. I suggest adding a `logging.warn` at the very least to give information to the user that certain values from the log were ignored.

Reply:
^